examples/website-demos/social-messaging/src/components/EmptyStateIndicator/SocialEmptyStateIndicator.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import { EmptyStateIndicatorProps } from 'stream-chat-react';
+
+import './SocialEmptyStateIndicator.scss';
+
+type Props = EmptyStateIndicatorProps;
+
+export const SocialEmptyStateIndicator: React.FC<Props> = (props) => {
```

Comment:
you can just put EmptyStateIndicatorProps here unless you're planning to extend them